src/modules/patient/providers/patient-query.service.ts

```diff
@@ -75,6 +76,13 @@ export class PatientQueryService extends TypeOrmQueryService<Patient> {
         return patient;
     }
 
+    async archiveOnePatient(id: number) {
+        const result = await this.repo
+            .softDelete(id);
+        console.log(result);
```

Comment:
remove comment. Add functionality to get all this patient assessment and soft delete them